roles/v2v.ci/tasks/miq_conversion_hosts_enable.yml

```diff
@@ -66,14 +67,19 @@
         ansible_ssh_pass: "{{ v2v_ci_rhv_root_password }}"
         ansible_pipelining: true
 
-    - name: Read RHV Engine SSH Private key
+    - name: Read RHV Engine SSH RSA Private key file /root/.ssh/id_rsa
```

Comment:
We will work with this key for now, as the ansible playbook, that created the conversion host VM was run from RHV engine, 
but later we'll need to replace it with key from the flow node, that will run this ansible playbook.